### PR TITLE
COMMERCE-11848 - "search x in category" text in commerce search dropdown is not translated

### PR DESCRIPTION
**Pull request by victorcdp**
Related Issue: [COMMERCE-11848](https://liferay.atlassian.net/browse/COMMERCE-11848)

### CODE REVIEW

**Comment by victorcdp** [comment]
ci:test:sf

**Comment by victorcdp** [comment]
@andrea-ale-sbarra Can you review this one? tied to an LPP. Thank you!

**Comment by victorcdp** [comment]
ci:test:relevant

**Comment by victorcdp** [comment]
ci:test:commerce

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -14738,6 +14738,7 @@ search-with-index-key-description=Leave this checked to use the index to search
 search-with-query-string-syntax=Search with Query String Syntax
 search-with-query-string-syntax-description=Use Query String query syntax to match one or more fields with a single operator (OR or AND).
 search-x=Search {0}
+search-x-in-x=Search "{0}" in {1}
```
Comment (brianikim) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:14741]: Why does {0} need the quotation marks here?

Comment (victorcdp) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:14741]: For consistency, the original string had quotation marks around the queryValue

Comment (brianikim) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:14741]: Thank you for clarifying.

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.

Thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by victorcdp** [comment]
ci:test:sf

**Comment by victorcdp** [comment]
ci:test:relevant

**Comment by victorcdp** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
QA Passed, add automation and resent on https://github.com/liferay-commerce/liferay-portal/pull/3975

**Comment by jaydawu** [comment]
ci:close

**Pull request closed without merging**